compiler/rustc_builtin_macros/src/source_util.rs

```diff
@@ -188,7 +188,7 @@ pub fn expand_include_str(
                 base::MacEager::expr(cx.expr_str(sp, interned_src))
             }
             Err(_) => {
-                cx.span_err(sp, format!("{} wasn't a utf-8 file", file.display()));
+                cx.span_err(sp, format!("{} wasn't an utf-8 file", file.display()));
```

Comment:
that's not right; the U in UTF is pronounced like you

Reply:
Thank you for your patient guidance, it has been fixed